hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/utils/HawkbitCommonUtil.java

```diff
@@ -574,14 +578,9 @@ public static String getArtifactoryDetailsLabelId(final String name, final Vaadi
      */
     public static Locale getLocaleToBeUsed(final UiProperties.Localization localizationProperties,
             final Locale desiredLocale) {
-        final List<String> availableLocals = localizationProperties.getAvailableLocals();
-        // ckeck if language code of UI locale matches an available local.
-        // Country, region and variant are ignored. "availableLocals" must only
-        // contain language codes without country or other extensions.
-        if (availableLocals.contains(desiredLocale.getLanguage())) {
-            return desiredLocale;
-        }
-        return new Locale(localizationProperties.getDefaultLocal());
+        return localizationProperties.getDefaultLocal() == null
+                ? desiredLocale
+                : localizationProperties.getDefaultLocal();
```

Comment:
Doesn't this change the logic, because before the browser-locale had the highest priority.
Would it be possible so set fallbackToSystemLocale=false and completely rely on spring's MessageSource-management. Then that the code and properties used to determine the locale would become obsolete?